com.unity.multiplayer.mlapi/Runtime/Transports/Transport.cs

```diff
@@ -109,6 +109,11 @@ public TransportChannel[] MLAPI_CHANNELS
             {
                 Name = "MLAPI_TIME_SYNC",
                 Type = ChannelType.Unreliable
+            },
+            new TransportChannel()
+            {
+                Name = "STDRPC",
+                Type = ChannelType.ReliableSequenced
```

Comment:
is this really needed? IIRC, it was working fine even without putting this channel definition here but I might be wrong.

Reply:
This was not part of my changes, so I am not sure why this is new.